README.md

```diff
@@ -133,7 +133,7 @@ Contributions are always welcome, but please abide by our community [code of con
 
 To contribute potential features or bug-fixes:
 
-1. Fork this repository
+1. Fork this repository, and clone with submodules: `git clone --recurse-submodules -j8 git@github.com:thunder-app/thunder.git`
```

Comment:
@hjiangsu For people who have already cloned the repo, they may have to run a command like this to init the submodule.

```
git submodule update --init packages/push
```